_episodes/02-running-containers.md

```diff
@@ -180,4 +180,12 @@ with Singularity available.
 > {: .solution}
 {: .challenge}
 
+## Exiting a singularity image
+
+The `exit` command exits a singularity instance. Note that when exiting from the singularity image all the running processes are killed (stopped). However, the work that you have done within the image is saved.
```

Comment:
Not exactly, you need to run/open a shell using `--writable` to preserve the changes.

Reply:
I would not mention anything about changes yet in this chapter, it may be confusing without talking about binds. This we will discuss later (chapter 7).

Reply:
sure I will remove the last sentence thanks